README.md

```diff
@@ -200,6 +203,39 @@ Example:
 ```
 mvn versions:set -DnewVersion=0.1.1-RC -DgenerateBackupPoms=false
 ```
+## Apply modification on Gnirps
+The following steps will allow you to make changes and integrate them into the project using Gnirps.
+
+### Setup on Gnirps
+- First, you need to set up a [new version](#versioning) of Gnirps.
+- Make your modification.
+- Install the main package in your local repository with the following command:
+```
+mvn clean install -N
+```
+- Install all the sub package in your local repository with the following command:
+```
+mvn clean install
+```
+- Create a distributable format 
+```
+mvn package
+```
+- Make sure you have, in your `~/.m2/repository/com/gnirps` folder, the `<<NEW_GNIRPS_VERSION>>-RC` folder on each submodule folder.
+
+### Use the new version in your local project
+- Specify the new Gnirps version in your `pom.xml` of your local project.
+- In intellij, you need to reload all maven project then use the following command:
+```
+mvn clean install
+```
+**_NOTE:_**  If you have some trouble with maven trying to find the newest version online and not in your `.m2` folder, comment the following line in your pom.xml:
```

Comment:
I'm not sure to follow. What's the issue you're trying to work around?

`mvn --update-snapshots` will force a check of the hash of the snapshots of your local repository against the remote one, and download a new version if the hashes doesn't match. Might that be what you're trying to do here?